### ISSUE TITLE
splparser format SQL syntax error

### ISSUE THREAD

**Issue by ghost** [vitessio/vitess#3773]
Hello and me again:

I found this problem with sqlparser: 
```
Input: rename table tb to t;
Output: rename table tb t;
```

This syntax has an error，the 'to' was lost here.

```
ERROR 1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 't' at line 1
```

**Comment by yuananf** [comment]
Should be fixed by #3774

**Comment by LepX** [comment]
emm ... This is another question:
```
Input: (select 1);
Output: syntax error at position 12
```

**Comment by martianzhang** [comment]
“(select 1);”
This may be another synax parser problem.
In my opinion following way should fix this problem
```c
--- a/go/vt/sqlparser/sql.y
+++ b/go/vt/sqlparser/sql.y
@@ -189,7 +189,7 @@ func forceEOF(yylex interface{}) {
 %token <bytes> UNUSED

 %type <statement> command
-%type <selStmt> select_statement base_select union_lhs union_rhs
+%type <selStmt> select_statement base_select union_lhs union_rhs paren_select_statement
 %type <statement> stream_statement insert_statement update_statement delete_statement set_statement
 %type <statement> create_statement alter_statement rename_statement drop_statement truncate_statement
 %type <ddl> create_table_prefix
@@ -297,6 +297,10 @@ command:
   {
     $$ = $1
   }
+| paren_select_statement
+  {
+    $$ = $1
+  }
 | stream_statement
 | insert_statement
 | update_statement
@@ -333,6 +337,16 @@ select_statement:
     $$ = &Select{Comments: Comments($2), Cache: $3, SelectExprs: SelectExprs{Nextval{Expr: $5}}, From: TableExprs{&AliasedTableExpr{Expr: $7}}}
   }

+paren_select_statement:
+  openb select_statement closeb
+  {
+    $$ = $2
+  }
+| openb paren_select_statement closeb
+  {
+    $$ = $2
+  }
+
```

**Comment by sougou** [comment]
Just fixing the parser won't be sufficient. The preview function also needs to be fixed.

Actually, we have to deprecate `Preview` and refactor vtgate to rely only on the parser. Hopefully, that refactor will also address the multi-statement problem.

I'll try to squeeze time out for this.

**Comment by zr-hebo** [comment]
hello, boy. We resolve this problem too, as mentioned before, do not deal with Preview yet. We have used in product.

![screenshot from 2018-03-24 16-51-50](https://user-images.githubusercontent.com/26314914/37862188-bed661d0-2f83-11e8-9cb2-d9ec050af999.png)

later, we will commit it if needed.

**Comment by harshit-gangal** [comment]
This is fixed with #6342
